docs/rst/introduction/developer_tools.rst

```diff
@@ -20,7 +20,7 @@ Developer Tools
 - :ref:`ROS 2 Monitor <vulcanexus_monitor>`: graphical desktop application to monitor *ROS 2* communications.
 - :ref:`Fast DDS Statistics Backend <fastdds_statistics_backend>`: *ROS 2 Monitor*'s backend. Can be leveraged with other monitoring frontend applications (e.g. :ref:`Prometheus <tutorials_tools_prometheus>`).
 - :ref:`ROS 2 Shapes Demo <vulcanexus_shapes_demo>`: first demo application to understand the most used *ROS 2* Quality of Service (QoS) and test DDS and *ROS 2* communication.
-- :ref:`micro-ROS <tutorials_micro_introduction_microros>`: default embedded toolkit for *ROS 2* and *Vulcanexus*.
+- :ref:`micro-ROS <vulcanexus_microros>`: default embedded toolkit for *ROS 2* and *Vulcanexus*.
```

Comment:
I wonder whether microros is a developer tool

Reply:
Removed here: 8574e5be1f171fec4dc0e2d2026afb0abe1a32dc